doc/mapping/full-text-search.md

```diff
@@ -2,9 +2,149 @@
 
 PostgreSQL has [built-in support for full-text search](https://www.postgresql.org/docs/current/static/textsearch.html), which allows you to conveniently and efficiently query natural language documents.
 
-*TO BE COMPLETED*
+## Mapping
+PostgreSQL full text search types are mapped onto C# types built-in to Npgsql. The ```tsvector``` type is mapped to ```NpgsqlTsVector``` and ```tsquery``` is mapped to ```NpgsqlTsQuery```. This means you can use properties of type ```NpgsqlTsVector``` directly in your model to create ```tsvector``` columns. The ```NpgsqlTsQuery``` type on the other hand, is used in LINQ queries.
```

Comment:
Maybe .NET types?

FYI, you can use single backticks to make inline code blocks (: